scripts/utility-vm/Makefile

```diff
@@ -0,0 +1,41 @@
+SOURCE_IMAGE=rhel-guest-image.qcow2
+TMP_DISK=disk.qcow2.tmp
```

Comment:
Is this gonna be created on provider? If yes, can this be renamed to `rhel-utility-vm-temp-disk.qcow`?

Reply:
this is created locally

Reply:
@kedark3 I don't think this is wise. This is a temporary file, not a disk for the temp partition. This disq.qcow2.tmp file is renamed to snapshot.qcow2 at line 17 and then to utility_vm.qcow2 ism ade from it at step on line 5